src/Protocol/Protocol_1_10.cpp

```diff
@@ -558,6 +558,83 @@ void cProtocol_1_10_0::WriteEntityMetadata(cPacketizer & a_Pkt, const cEntity &
 			break;
 		}  // case etItemFrame
 
+		case cEntity::etArmorStand:
+		{
+			auto & ArmorStand = static_cast<const cArmorStand &>(a_Entity);
+
+			if (ArmorStand.HasCustomName())
+			{
+				a_Pkt.WriteBEUInt8(ENTITY_CUSTOM_NAME);
+				a_Pkt.WriteBEUInt8(METADATA_TYPE_STRING);
+				a_Pkt.WriteString(ArmorStand.GetCustomName());  // Must be set as alwaysVisible to be visible (idk why)
+
+				a_Pkt.WriteBEUInt8(ENTITY_CUSTOM_NAME_VISIBLE);
+				a_Pkt.WriteBEUInt8(METADATA_TYPE_BOOL);
+				a_Pkt.WriteBool(ArmorStand.IsCustomNameAlwaysVisible());
+			}
+
+			Flags = 0;  // Armor stand has its own flags
+			if (ArmorStand.IsSmall())
+			{
+				Flags |= 0x01;
+			}
+			if (ArmorStand.HasArms())
+			{
+				Flags |= 0x04;
+			}
+			if (!ArmorStand.HasBasePlate())
+			{
+				Flags |= 0x08;
+			}
+			if (ArmorStand.IsMarker())
+			{
+				Flags |= 0x10;
+			}
+			a_Pkt.WriteBEUInt8(ARMOR_STAND_STATUS);  // Index
+			a_Pkt.WriteBEUInt8(METADATA_TYPE_BYTE);  // Type
+			a_Pkt.WriteBEInt8(Flags);
+
+			if (!ArmorStand.IsMarker())  // No point sending orientation if it's a marker. However invisible armor stand keep armor visible.
```

Comment:
Why would markers not need to broadcast their orientation? You can still see them and pose them.

Reply:
Like shown in https://youtu.be/Vgc_tZ2GHvg?t=67
Invisible armor stand are not so invisible (confusing tag name)
At the opposite, armor stand sets as marker are really and totally invisible. The only part still shown is the custom name if sets as always visible.

Then sending armor stand's body rotation is still useful if the armor stand has an armor.
However an armor stand sets as marker will not be shown at all, then theses packets are not useful in this case.

Else if I am wrong ?

Reply:
When I try out your code, I can see the markers though? Either way, something is wrong here.

Reply:
This discussion shows that a proper comment in the code is needed to explain this.

Reply:
I updated it, wait the push to see if it's OK